src/sentry/models/apikey.py

```diff
@@ -28,7 +28,7 @@ class ApiKeyStatus:
 class ApiKey(ReplicatedControlModel, HasApiScopes):
     __relocation_scope__ = RelocationScope.Global
     category = OutboxCategory.API_KEY_UPDATE
-    replication_version = 2
+    replication_version = 3
```

Comment:
Intended to bump this in a previous PR.  Doing so will make it possible for me to trigger a backfill via options.